server/src/controllers/user.controller.ts

```diff
@@ -43,6 +44,22 @@ export class UserController {
     return this.service.search(auth);
   }
 
+  @Get('me/activity')
+  @Authenticated({ permission: Permission.UserRead })
+  async getMyUploadActivity(
+    @Auth() auth: AuthDto,
+    @Query('from') from?: string,
```

Comment:
these two `from` and `to` should be coming from a DTO object for data validation